docs/technical_specs/model_libraries/power_generation/costing/power_plant_costing.rst

```diff
@@ -11,8 +11,14 @@ Introduction
 
 A capital cost methodology is developed in this module, both bare and erected cost and total plant cost are calculated based on costing correlations. 
 The Power Plant Costing Library contains two main costing functions `get_PP_costing` and `get_SCO2_unit_cost`.
-The first function (get_PP_costing) can be called to include cost correlations for equipment typically used in simulation of 7 technologies: supercritical pulverized coal plants (SCPC),
-subcritical pulverized coal plants, Two-stage IGCC, Single-stage IGCC, Single-stage dry-feed IGCC, natural gas air-fired plant (NGCC), and advanced ultra-supercritical PC (AUSC).
+The first function (get_PP_costing) can be called to include cost correlations for equipment typically used in simulation of 7 technologies: 
+
+1. Supercritical pulverized coal plants (SCPC),
+2. Subcritical pulverized coal plants,
+3. Two-stage IGCC, Single-stage IGCC,
+4. Single-stage dry-feed IGCC, natural gas air-fired plant (NGCC),
+5. Advanced ultra-supercritical PC (AUSC).
```

Comment:
Since you say seven, probably should have 7 items in the list.  I think a couple lines may have two items.